### PR TITLE
Portfolio: details popup window

### PR DESCRIPTION
**Pull request by lfmnovaes**
### Project requirements

- You should implement the following interactions:
  - When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
  - When the user clicks (or taps) the close (X) button, the popup disappears.
- You should implement popups for both mobile and desktop screen sizes:

We included a screenshot of Template 5 for reference, but you should follow the template that you chose.
- You must stick to the design as much as possible (e.g., font, colors, images, tests, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
- In order to associate each project with the popup window details, you must use **a JavaScript array to store all of the information for all projects**.
  - For each project you need to store the following pieces of data in a JavaScript object, at a minimum: name, description, featured image, technologies, link to live version, link to source.
  - You need to update the main page so that the projects section is created dynamically using the information stored in that JavaScript object. Remember, all of the HTML in that section is created when the page loads.
  - After that, you could implement the popup window.

- **You need to use pair-programming for this project. We will check to make sure that both partners have contributed to the project by looking at the commit history for each project. Therefore you need to implement the above features in all of the portfolios in your pair-programming group.**

[Source](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md)

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello  

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._


Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -105,7 +105,38 @@ <h2 class="proj-title bold font40-52">Tonic</h2>
...
+        </div>
+      </div>
+      <div id="myModal1" class="modal">
+        <div class="modal-container">
+          <div class="modal-content">
+            <div class="modal-title">
+              <h1 class="proj-title bold font40-52">Tonic</h1>
+              <span class="xp1">&times;</span>
+            </div>
+            <div class="proj-subt d-flex modal-subt">
```
Comment (mikenath223) [inline index.html]: - [ ] As requested in the [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md#:~:text=you%20must%20use%20a%20JavaScript%20array%20to%20store%20all%20of%20the%20information%20for%20all%20projects.), kindly represent all projects as a JavaScript array of projects and dynamically render the project from the array.

File: js/scripts.js [hunk 2]
```diff
@@ -3,21 +3,41 @@ const ham = document.querySelector('.ham');
...
+      proj.style.filter = 'blur(4px)';
+    },
+  );
+}
+
+function projOff() {
+  projs.forEach(
+    (proj) => {
+      proj.style.filter = 'blur(0px)';
+    },
```
Comment (mikenath223) [inline js/scripts.js:22]: - [ ] Kindly change these and other related functions to es6 arrow functions, as required in the JavaScript best practices section.

Comment (lfmnovaes) [reply js/scripts.js:22]: Hello @mikenath223 , I tried to do what you told me to switch all "normal" functions to ES6 arrow functions and while everything was working fine, I started to get a LOT of ESLint 'not defined' function errors. Seems like ESLint doesn't detect arrow functions.

**Review by karvel-code — APPROVED** [review]
Hi @lfmnovaes , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**